bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -21,12 +29,27 @@ spec:
       annotations:
         target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
       labels:
+        {{ if eq .OVN_DPU_MODE "dpu-host" }}
+        app: ovnkube-node-dpu-host
+        {{ else }}
         app: ovnkube-node
+        {{ end }}
         component: network
         type: infra
         openshift.io/component: network
         kubernetes.io/os: "linux"
     spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
note that (as the overly-long name indicates), this only affects the initial scheduling; it will not cause existing `ovnkube-node` pods on `dpu-host` nodes to be killed

Reply:
@danwinship  Yes, I think we shall change to use requiredDuringExecution when it is supported in k8s.
From my testing, removing the label `dpu-host` also triggers the removal of `ovnkube-node-dpu-host` pods. Could it be due to the reconciling of daemonset changes the resourceVersion which is considered as a new scheduling event?